xlml/utils/tpu.py

```diff
@@ -88,6 +89,13 @@ def create_queued_resource_request(tpu_name: str, ssh_keys: ssh.SshKeys) -> str:
           node_count=num_slices, node_id_prefix=tpu_name
       )
 
+    metadata = {
+        'ssh-keys': f'ml-auto-solutions:{ssh_keys.public}',
+    }
+
+    if startup_script is not None:
+      metadata['startup-script'] = f'{startup_script}'
```

Comment:
Can we have the default empty value for the `startup_script` so that we don't have this if statement and logic can be merged into the line 92?

Reply:
Done